engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/views/activation-key-info.html

```diff
@@ -6,6 +6,12 @@
   </span>
 </div>
 
+<div bst-alert="warning" ng-if="activationKey.multi_content_view_environment">
+  <span translate>
+    This activation key has multiple content view environments, which are not yet displayed in the web UI. Changing the content view or lifecycle environment here will overwrite the activation key's multiple environments. To avoid this, update the activation key via Hammer.</a>
```

Comment:
Should we give an example of the hammer command they should use like:
```bash
hammer activation-key update --id=xxx --organization-id=x --content-view-environments='env_label1/cv_label1,env_label2/cv_label2
```

Reply:
Ehhh, I think users can use hammer help :)